src/main/java/com/bunq/sdk/http/ApiClient.java

```diff
@@ -53,6 +54,18 @@
 public class ApiClient {
 
   /**
+   * Endpoints not requiring active session for the request to succeed.
+   */
+  private static final String DEVICE_SERVER_URL = "device-server";
+  private static final String INSTALLATION_URL = "installation";
+  private static final String SESSION_SERVER_URL = "session-server";
+  private static List<String> URIS_NOT_REQUIRING_ACTIVE_SESSION = Arrays.asList(
+      DEVICE_SERVER_URL,
+      INSTALLATION_URL,
+      SESSION_SERVER_URL
+      );
```

Comment:
Is it the formatting that IDE suggested? I mean I would expect there something like:
```
URIS_NOT_REQUIRING_ACTIVE_SESSION = Arrays.asList(
    DEVICE_SERVER_URL,
    INSTALLATION_URL,
    SESSION_SERVER_URL
);
```

Reply:
Also, please make it `private static` final! `asList` allows for it if I am not mistaken :)